ios/App/App/Info.plist

```diff
@@ -83,5 +83,10 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<true/>
+  <key>NSAppTransportSecurity</key>
+  <dict>
+  <key>NSAllowsArbitraryLoads</key>
```

Comment:
what is this needed for (http urls?). I assume this will this be less secure? 
If required i would like this to be configurable per project like we do in the pRMT app through remote config (by default it should be secure option)

Reply:
Ah yes I was just copying this from the `dev` branch because I thought it was needed. Removed now.